content/en/observability_pipelines/_index.md

```diff
@@ -46,6 +46,9 @@ The Datadog UI provides a control plane to manage your Observability Pipelines W
 
 1. [Set up the Observability Pipelines Worker][1].
 2. [Create pipelines to collect, transform and route your data][2].
+3. Deep dive into recommendations for deploying and scaling multipe Observability Pipelines Workers:
+    - See [Deployment Design and Principles][3] for information on what to consider when designing your Observability Pipelines architecture.
+    - See [Best Practices for OPW Aggregator Architecture][4] for details on the recommended Observability Pipelines aggregator architecture, which is optimized for scaling.
```

Comment:
This repeats itself a bit (or seems wordy to me), what about: "See [Best Practices for OPW Aggregator Architecture][4] for recommended Observability Pipelines architecture, optimized for scaling."

Just brainstorming, but it is ok as-is too.

Reply:
Or we could just cut it down to:

> See [Best Practices for OPW Aggregator Architecture][4].

The title does explain itself :).